tools/slicec-cs/src/decoding.rs

```diff
@@ -75,7 +73,7 @@ pub fn default_activator(encoding: Encoding) -> &'static str {
 fn decode_member(member: &impl Member, namespace: &str, encoding: Encoding) -> CodeBlock {
     let mut code = CodeBlock::default();
     let data_type = member.data_type();
-    let type_string = data_type.cs_type_string(namespace, TypeContext::IncomingParam, true);
+    let type_string = data_type.incoming_parameter_type_string(namespace, true);
```

Comment:
Should we rename this function "decode_parameter"?

Reply:
No, this function is used for both fields and parameters.
But the fact we were passing `IncomingParam` is actually bogus.
We only use the `type_string` variable for structs and classes... which don't have a special mapping as parameters.